test/bin/mdx-test/expect/ocaml-errors/test-case.md

```diff
@@ -37,14 +37,14 @@ end
 
 Existing error blocks attached to a valid ocaml block are removed:
 
-```ocaml
+```ocaml version>=5.2
```

Comment:
This block shouldn't be guarded, the error message is actually removed and not checked.
```suggestion
```ocaml
```